api/types/plugin.go

```diff
@@ -127,6 +128,12 @@ type PluginArgs struct {
 	Value []string
 }
 
+// PluginKernelModule represents a Loadable Kernel Modules (LKM) to modprobe before enabling a plugin
+type PluginKernelModule struct {
+	PluginSetting
```

Comment:
it's never used for now right ? my understanding was at some point `docker plugin set` will allow the user to change the modules, but maybe it shouldnt ?